scripts/seed.sql

```diff
@@ -1,38 +1,125 @@
 USE askgov;
 
-INSERT INTO tags(tagname, description, tagType, link, hasPilot, createdAt, updatedAt)
+INSERT INTO `askgov`.`tags` (`id`, `tagname`, `description`, `link`, `hasPilot`, `createdAt`, `updatedAt`, `tagType`)
 VALUES
-("mci", "Ministry of Communications and Information", "AGENCY", "https://www.mci.gov.sg/contact-info", true, "2021-01-11 11:54:06", "2021-01-11 11:54:06"),
-("mccy", "Ministry of Culture, Community, and Youth", "AGENCY", "https://www.mccy.gov.sg/contact-us", true, "2021-01-11 11:54:06", "2021-01-11 11:54:06"),
-("ogp", "Open Government Products", "AGENCY", "https://open.gov.sg", true, "2021-01-11 11:54:06", "2021-01-11 11:54:06"),
-("govtech", "Government Technology Agency", "AGENCY", "https://www.tech.gov.sg/", true, "2021-01-11 11:54:06", "2021-01-11 11:54:06");
+(5, 'was', 'Work Allocation Singapore', 'https://www.was.gov.sg/', 1, NOW(), NOW(), 'AGENCY'),
+(6, 'financial support', 'Financial Support', '', 0, NOW(), NOW(), 'TOPIC'),
+(7, 'apprenticeships', 'Apprenticeships', '', 0, NOW(), NOW(), 'TOPIC'),
+(8, 'employers', 'Employers', '', 0, NOW(), NOW(), 'TOPIC'),
+(9, 'awards', 'Awards', '', 0, NOW(), NOW(), 'TOPIC')
+;
 
-INSERT INTO agencies(shortname, longname, logo, createdAt, updatedAt)
+INSERT INTO `askgov`.`agencies` (`id`, `shortname`, `longname`, `logo`, `email`, `createdAt`, `updatedAt`)
```

Comment:
`https://logos.ask.gov.sg/askgov-logo.png` gives access denied

Reply:
Deliberate - I have yet to upload the image